server/services/lifecycle/lifecycle.js

```diff
@@ -72,10 +72,39 @@ module.exports = ({ strapi }) => {
               )
             })
         },
-        async afterUpdateMany() {
-          strapi.log.error(
-            `Meilisearch could not find an example on how to access the \`afterUpdateMany\` hook. Please consider making an issue to explain your use case`
-          )
+        async afterUpdateMany(event) {
```

Comment:
Is there a way to test this?

Reply:
Unfortunately it would be very hard to test because of the structure of the subscription. 
In the previous version of Strapi v3 I used to try it in the cypress tests, but these are on their way: 

#351
#103